### PR TITLE
New game - user can click new and create a new game obj

### PR DESCRIPTION
**Pull request by brij8**
wordlist-new-game

### CODE REVIEW

File: client/pages/games.jsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      games: [],
+      lists: []
+    };
+    this.newGame = this.newGame.bind(this);
+    this.getGame = this.getGame.bind(this);
+    this.getWords = this.getWords.bind(this);
+  }
+
+  componentDidMount() {
+    fetch('api/games')
```
Comment (scbowler) [inline client/pages/games.jsx]: All of your `fetch` request paths should start with a `/` so this one should be `/api/games`. While this works okay in development, without the slash it is likely to break in production.

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, only update needed is to add a starting `/` to all of your `fetch` requests. Let me know if you have any questions.

File: client/pages/games.jsx (hunk 1 above)
Comment (brij8) [reply client/pages/games.jsx]: got it, on it 👍

File: client/pages/home.jsx [hunk 2]
```diff
@@ -1,8 +1,28 @@
...
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      currentGame: 'gameName'
+    };
+  }
+
+  componentDidMount() {
+    fetch('api/home')
```
Comment (scbowler) [inline client/pages/home.jsx]: Missed this one

File: client/pages/lists.jsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      lists: []
+    };
+  }
+
+  componentDidMount() {
+    fetch('api/lists')
```
Comment (scbowler) [inline client/pages/lists.jsx]: Here

**Review by scbowler — CHANGES_REQUESTED** [review]
Getting closer, missed to `fetch` requests

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**